apps/vr-tests-react-components/src/utilities/utilities.test.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { getStoryVariant, DARK_MODE, HIGH_CONTRAST, RTL } from './getStoryVariant';
+import { Button } from '@fluentui/react-button';
+
+describe('utility functions', () => {
```

Comment:
can we add also test for that `storyName` which uses that regexp ?

Reply:
[781e403](https://github.com/microsoft/fluentui/pull/25108/commits/781e403f9b5618858c9ce9104437d949e202b994)